_topic_maps/_topic_map.yml

```diff
@@ -3270,6 +3270,22 @@ Topics:
       File: telco-core-ref-crs
     - Name: Telco core software specifications
       File: telco-core-ref-software-artifacts
+- Name: Comparing cluster configurations
+  Dir: cluster-compare
+  Distros: openshift-origin,openshift-enterprise
+  Topics:
+  - Name: Understanding the cluster-compare plugin
+    File: understanding-the-cluster-compare-plugin
+  - Name: Installing the cluster-compare plugin
+    File: installing-cluster-compare-plugin
+  - Name: Using the cluster-compare plugin
+    File: using-the-cluster-compare-plugin
+  - Name: Creating a reference configuration
```

Comment:
Since `You have access to a reference configuration` is listed as a prerequisite for `Using the cluster-compare plugin`, would it make sense to have the `Creating a reference configuration` document listed before the `Using the cluster-compare plugin` document? I think it may flow better for people who are following along step-by-step.

Reply:
Good question. Typically, after some conceptual info, it's good to follow with the most basic use-case so that the user can start road-testing the feature, and then adding complexity. So that was the reason why "Using the plugin.." came first. 

And also, I think the reference configuration is provided by Red Hat to the user. I think that would be the typical workflow - at least right now - rather than the user beginning by creating their own configuration and then running the plugin. 

What do you think @lack ?